src/citation/views/citation_entry_view.py

```diff
@@ -44,22 +44,60 @@ def retrieve(self, request):
 
     @action(detail=False, methods=["post"], permission_classes=[IsAuthenticated])
     def pdf_uploads(self, request):
```

Comment:
Minor comment: Can we change to `upload_pdf`, `upload_files` or `upload_pdfs` ? I think having a verb in the name of the action would be better

Reply:
Will do